tests/EdFi.Common.UnitTests/Security/Pbkdf2HmacSha1SecureHasherTests.cs

```diff
@@ -24,39 +26,36 @@ public class When_computing_a_valid_hash : TestFixtureBase
         {
```

Comment:
As mentioned earlier, doesn't appear to be a test covering a null salt for a valid hash algorithm code.  Is that scenario expected?

Reply:
i dont think so, but i will add a test